### PR TITLE
Have a look 

### PR DESCRIPTION
**Pull request by StuartIanNaylor**
I left the questions about the rsync commands but rsync is a much better way due to the --exclude argument

Slightly confused as the options you seem to have repeats just in long & short form or already implicit by -a

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -4,22 +4,43 @@ systemctl -q is-active log2ram  && { echo "ERROR: log2ram service is still runni
...
+install -m 644 log2ram.conf /usr/local/bin/log2ram/log2ram-lastcheck
+install -m 644 uninstall.sh /usr/local/bin/log2ram/uninstall-log2ram.sh
 systemctl enable log2ram
 
 # cron
 install -m 755 log2ram.hourly /etc/cron.hourly/log2ram
 install -m 644 log2ram.logrotate /etc/logrotate.d/log2ram
 
 # Remove a previous log2ram version
-  rm -rf /var/log.hdd
```
Comment (azlux) [inline install.sh]: Don't change that. It's when you upgrade from a old version of log2ram

File: log2ram.conf [hunk 2]
```diff
@@ -14,12 +14,12 @@ USE_RSYNC=false
 
 # If there are some errors with available RAM space, a system mail will be send
 # Change it to false and you will have only a log if there is no place on RAM anymore.
-MAIL=true
+MAIL=false
```
Comment (azlux) [inline log2ram.conf]: Don't change the default

File: log2ram.conf [hunk 3]
```diff
@@ -14,12 +14,12 @@ USE_RSYNC=false
...
 
 # **************** Zram backing conf  *************************************************
 
 # ZL2R Zram Log 2 Ram enables a zram drive when ZL2R=true ZL2R=false is mem only tmpfs
-ZL2R=false
+ZL2R=true
```
Comment (azlux) [inline log2ram.conf]: Don't change the default

Comment (StuartIanNaylor) [reply log2ram.conf]: I have no idea why you are even commenting? I said give that a try?
IE give it a try not pull / push or anything.

I can change what I wish when I am demonstrating something especially the items which need to be seen for the demonstration.

Still haven't answered the important questions about writes or the start up that breaks applications expecting log files?

File: log2ram.conf (hunk 2 above)
Comment (StuartIanNaylor) [reply log2ram.conf]: The mail option for default needs to go its quite strange to expect a mail install such as postfix on limited hardware that would employ L2R.
But as said later I am asking about writes and the whole sync to disk routine that likely creates as much block wear as logging to flash.
Its dependent on frequency of logging  but with the majority of expected applications it highly likely L2R is currently providing little benefit.
It writes out every out complete files for no reason and it doesn't matter about conf settings apart they are the conf settings I am asking you to give a go?

File: install.sh [hunk 4]
```diff
@@ -1,24 +1,20 @@
-#!/usr/bin/env sh
+#!/bin/bash
```
Comment (azlux) [inline install.sh]: No. This is not a recommend way to use bash

**Pull request closed without merging**